clang/lib/Basic/Targets/PPC.cpp

```diff
@@ -439,14 +443,12 @@ void PPCTargetInfo::getTargetDefines(const LangOptions &Opts,
   // FIXME: The following are not yet generated here by Clang, but are
   //        generated by GCC:
   //
-  //   _SOFT_FLOAT_
   //   __RECIP_PRECISION__
   //   __APPLE_ALTIVEC__
   //   __RECIP__
   //   __RECIPF__
   //   __RSQRTE__
   //   __RSQRTEF__
```

Comment:
Would you please also handle `__RSQRTE__` and `__RSQRTEF__`? These two macros should also be impacted by soft-float. Thanks.

Reply:
Looks like `__RECIP__` and `__RECIPF__` too? Will do.

Reply:
Ah, no, I guess Clang doesn't have those builtins yet, so that wouldn't be right. I'll just do the two you mentioned.